.changes/window-state-physical-size.md

```diff
@@ -0,0 +1,5 @@
+---
+"window-state": patch:breaking
+---
+
+Window's size is now stored in physical size instead of logical size
```

Comment:
is there a reason why this was changed?

Reply:
We store physical position, and this is to align with it, and physical size makes more sense to me, if I open a window that occupies 75% of the screen on 100% DPI, and later on I changed the DPI to 125%, I would like it to still occupy 75% of the screen (this can be debated either ways)

Reply:
isn't this what logical is supposed to do? i.e. if your window size is 75% of the monitor logical size, it should stay at 75% on a different monitor with the same resolution even if it was different DPI?

Reply:
say we have a 1080 height screen, first we store it on 100% DPI, which is 1080 * 0.75 = 810, and then on the 125% screen, it will be converted to 810 * 1.25 = 1012.5

Reply:
3 years and I still confuse these two. Thanks for explaining